signing_service/signing_service/signing_service.py

```diff
@@ -384,6 +410,28 @@ def _get_authentication_challenge_signature(self, authentication_challenge: byte
             authentication_challenge,
         )
 
+    @staticmethod
+    def _get_signing_service_daily_transaction_sum_so_far() -> int:
+        thresholds_directory = (Path.cwd()).joinpath('daily_tresholds')  # pylint: disable=no-member
+        thresholds_directory.mkdir(exist_ok=True)  # pylint: disable=no-member
+        daily_file = datetime.datetime.now().strftime('%Y-%m-%d')
+        thresholds_directory.joinpath(daily_file).touch(exist_ok=True)  # pylint: disable=no-member
+        thresholds_directory = thresholds_directory.joinpath(daily_file)
+        if thresholds_directory.read_text() == '':  # pylint: disable=no-member
+            thresholds_directory.write_text('0')  # pylint: disable=no-member
+            return 0
+        try:
+            transaction_sum = int(thresholds_directory.read_text())  # pylint: disable=no-member
+        except ValueError:
+            thresholds_directory.write_text('0')  # pylint: disable=no-member
+            return 0
+        return transaction_sum
+
+    @staticmethod
+    def _add_payload_value_to_daily_transactions_sum(transactions_sum: int) -> None:
+        thresholds_directory = (Path.cwd()).joinpath('daily_tresholds').joinpath(datetime.datetime.now().strftime('%Y-%m-%d'))
+        thresholds_directory.write_text(str(transactions_sum))
```

Comment:
you are writing to a directory. I assume, we would like to create a file first ;)

Reply:
It is created in `_get_signing_service_daily_transaction_sum_so_far()`, that we always call before. I didn't want to copy same code. Or should I?

Reply:
Ok, I made separate function `_get_daily_transaction_threshold_file_directory()` and I'm using it in both `_get_signing_service_daily_transaction_sum_so_far` and `_add_payload_value_to_daily_transactions_sum`